hr_timesheet_auto_creation/README.rst

```diff
@@ -54,20 +56,10 @@ Maintainer
 
 This module is maintained by Elico Corporation.
 
-Elico Corp is an innovative actor in China, Hong-Kong and Singapore servicing
-well known international companies and as well as local mid-sized businesses.
-Since 2010, our seasoned Sino-European consultants have been providing full
-range Odoo services:
+Elico Corporation offers consulting services to implement open source management
+software in SMEs, with a strong involvement in quality of service.
 
-* Business consultancy for Gap analysis, BPM, operational work-flows review. 
```

Comment:
please use latest template

Reply:
@elicoidal I already have used latest template.
Correct me if I am wrong and send me a new template or it's content.